src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,42 @@
 package mate.jdbc;
 
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Driver;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.service.DriverService;
+import mate.jdbc.service.ManufacturerService;
+
 public class Main {
+    private static Injector injector = Injector.getInstance("mate.jdbc");
+
     public static void main(String[] args) {
-        // test your code here
+        DriverService driverService = (DriverService) injector.getInstance(DriverService.class);
+        Driver driver = new Driver();
+        driverService.getAll().forEach(System.out::println);
+        driver.setName("Petro Poroshenko");
+        driver.setLicenseNumber("7777777");
+        driverService.create(driver);
+        Driver driver1 = driverService.get(1L);
+        System.out.println(driver1);
+        driverService.getAll().forEach(System.out::println);
+        driver.setName("Petro Poroshenko Chocolate King");
+        driverService.update(driver);
+        driverService.getAll().forEach(System.out::println);
+        driverService.delete(1L);
+        driverService.getAll().forEach(System.out::println);
```

Comment:
It's better to separate logically grouped lines of code by empty blocks for the sake of readability and easier understanding either in the future or by other devs.